modules/ossm-rn-deprecated-features.adoc

```diff
@@ -0,0 +1,35 @@
+////
+Module included in the following assemblies:
+- servicemesh-release-notes.adoc
+////
+
+[id="ossm-deprecated-features_{context}"]
+////
+Description - Description of the any features (including technology previews) that have been removed from the product.  Write the description from a customer perspective, what UI elements, commands, or options are no longer available.
+Consequence or a recommended replacement - Description of what the customer can no longer do, and recommended replacement (if known).
+////
+= Deprecated features
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+
+== Deprecated features {ProductName} 1.1.5
+
+The following custom resources are deprecated in this release and will be removed in a future release.
+
+* `Policy` - The `Policy` resource is deprecated and will be removed in a future Istio release. Migrate to the `PeerAuthentication` resource.
+* `MeshPolicy` - The `MeshPolicy` resource is deprecated and will be removed in a future Istio release. Migrate to the `PeerAuthentication` resource.
+* `v1alpha1` RBAC API -The v1alpha1 RBAC policy is deprecated by the v1beta1 Authorization policy. RBAC (Role Based Access Control) defines `ServiceRole` and `ServiceRoleBinding` objects.
+** `ServiceRole`
+** `ServiceRoleBinding`
+* `RbacConfig` - `RbacConfig` implements the `ClusterRbacConfig` Custom Resource Definition for controlling Istio RBAC behavior.
+** `ClusterRbacConfig`
+** `ServiceMeshRbacPolicy`
```

Comment:
ServiceMeshRbacConfig. I'm not sure what this part is intending though, ClusterRbacConfig was replaced by ServiceMeshRbacConfig in SM 1.0

Reply:
That was listed on the Google doc from when we were discussing differences between 1.4 and 1.6.  Should I pull it completely or should it be revised somehow?

From that doc:
"Deprecated: v1alpha1 RBAC API (had been deprecated in 1.4 already, replaced by  v1beta1 AuthorizationPolicy)
ServiceRole
ServiceRoleBinding
RbacConfig
ClusterRbacConfig (ServiceMeshRbacPolicy)"

Reply:
@knrc and/or @rcernich , please check updated text here for me.